tests/fedora-str/overlay-git.yml

```diff
@@ -0,0 +1,40 @@
+---
+- hosts: localhost
+  tags:
+  - atomic
+  remote_user: root
+  tasks:
+  # Install the built RPM to the VM
+  - command: git rev-parse HEAD
+    register: git_rev_head
+    connection: local
+  - name: Copy locally built RPMs
+    synchronize: src=x86_64/ dest=/root/x86_64/ archive=yes
+  - shell: ostree admin unlock || true
```

Comment:
I'm not a big fan of using `admin unlock` for this, given that it's not how it's actually run on client systems. It also limits what tests can do (e.g. testing `admin unlock` itself, or rebooting). Have you considered building our own tree like in rpm-ostree?

Actually, it should work to just `rpm-ostree override replace` the RPMs, reset the ref and change the origin back to use `refspec`, right?

Reply:
Mmm.  I use `admin unlock` a lot for testing personally where I really explicitly don't want things to persist.  We'll clearly need `vmcheck` and `override replace` to be used as well though.

(I'd be cool down the line to have `override override replace --transient` or something which would be like more like `replace` + "livefs-on-overlay" but still underneath the ro mount).

Reply:
Since the previous installed tests code in `fah-prep.sh` (now deleted) also used `admin unlock` i'd vote we keep it for now - and build upon this first step by e.g. copying the a-h-t reboot playbook and using it, etc.